### PR TITLE
Following [CALCITE-3753] add log when aborting optimization due to VolcanoTimeoutException (Jiatao Tao)

### PR DESCRIPTION
**Pull request by Aaaaaaron**

### CODE REVIEW

**Comment by Aaaaaaron** [comment]
Current volcano planner may abort an optimization actively, it's critical path, worth a log.

File: core/src/main/java/org/apache/calcite/plan/volcano/VolcanoPlanner.java [hunk 1]
```diff
@@ -519,6 +519,7 @@ public RelNode findBestExp() {
         try {
           match.onMatch();
         } catch (VolcanoTimeoutException e) {
+          LOGGER.warn("Catch VolcanoTimeoutException, abort optimization.");
           root = canonize(root);
```
Comment (danny0405) [inline core/src/main/java/org/apache/calcite/plan/volcano/VolcanoPlanner.java:523]: How about "Volcano planning times out, cancels the subsequent optimization."

Comment (Aaaaaaron) [reply core/src/main/java/org/apache/calcite/plan/volcano/VolcanoPlanner.java:523]: > How about "Volcano planning times out, cancels the subsequent optimization."

Revised

**Comment by yanlin-Lynn** [comment]
Need to capitalize the first letter in commit log

**Pull request closed without merging**